newIDE/app/src/ObjectEditor/ObjectsEditorService.js

```diff
@@ -16,16 +19,26 @@ const ObjectsEditorService = {
     if (this.editorConfigurations[objectType]) {
       return this.editorConfigurations[objectType];
     }
-    if (!project.hasEventsBasedObject(objectType)) {
-      console.warn(
-        `Object with type ${objectType} has no editor configuration registered. Please use registerEditorConfiguration to register your editor.`
+    if (project.hasEventsBasedObject(objectType)) {
+      const objectMetadata = gd.MetadataProvider.getObjectMetadata(
+        gd.JsPlatform.get(),
+        objectType
       );
+      return this.getCustomObjectPropertiesEditor({
+        helpPagePath: objectMetadata.getHelpPath(),
+      });
```

Comment:
Do you think that the events-based object generation should register instead.

Reply:
That's a good question. What would be the pros/cons of each?

Reply:
It makes sense for extensions from Core and GDJS as them will stay the same for the process life-time.
But, as this list is static, we probably don't want to pollute it with extensions from projects.

Reply:
Yeah, probably we don't want to pollute it. The list exists because extensions from Core and GDJS are fixed. But if we can be dynamic, no reason to make things constrained/harder to use with a list.